sdk/identity/test-resources-post.ps1

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# IMPORTANT: Do not invoke this file directly. Please instead run eng/New-TestResources.ps1 from the repository root.
+
+param (
+  [hashtable] $DeploymentOutputs
+)
+
+# # If not Linux, skip this script.
```

Comment:
Did you intend to leave this commented out?

Reply:
Umm, let me check on that.